AoDDownloader/utils.py

```diff
@@ -44,6 +51,7 @@ def create_login(use_keyring: bool = True):
 "pip install dbus-python" might fix this problem.
 Or use --no-keyring to enter password every time and disable keyring""")
     config.write()
+    click.echo("Login erfolgreich.")
```

Comment:
They should not be in German, could you change all other occurrences too, please?